c#/Mandoline.Api.Examples/Client.Repl/GlobalSuppressions.cs

```diff
@@ -0,0 +1,35 @@
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.SelectionCommand.#ctor")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.SelectionCommand.CreateSelection~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.SelectionCommand.GetSelection~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.SelectionCommand.UpdateSelection~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.ShapedDownloadCommand.#ctor")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.ShapedDownloadCommand.ShapedDownload~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.ShapedDownloadCommand.ShapedDownloadStream~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.UserCommand.#ctor")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.UserCommand.GetUser~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.UserCommand.GetUsers~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "member", Target = "~M:Client.Repl.UserCommand.LogIn~System.Threading.Tasks.Task")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "type", Target = "~T:Client.Repl.ConsoleOutput")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "<Pending>", Scope = "type", Target = "~T:Client.Repl.DownloadCommand")]
```

Comment:
I have suppressed most of the documentation messages for this PR as I am not sure what our documentation procedure is, as you can see there are a lot of missing doc comments. If these need to be addressed it may be better to do a new PR. Let me know what you guys think.